Luminary099/R30.agc

```diff
@@ -453,10 +454,10 @@ SKIPTPER	STODL	-TPER
 
```

Comment:
![image](https://user-images.githubusercontent.com/3440094/95390335-21118600-08ed-11eb-84d4-0ad4f9bfe023.png)
3 lines above:

- Extra 2 spaces before `HYPERMIN + RPADTEM`
- Single space between `RPADTEM.` and `(TPER`